File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist5.cs
// <snippetdistributecampaignfrommarketinglist5>


 // Retrieve the members that were distributed to.
 var retrieveMembersRequest = new RetrieveMembersBulkOperationRequest
 {
     BulkOperationId = distributeResponse.BulkOperationId,
     BulkOperationSource = (int)BulkOperationSource.CampaignActivity,
     EntitySource = (int)EntitySource.Contact,
     Query = new QueryExpression(Contact.EntityLogicalName)
 };

 var retrieveMembersResponse = (RetrieveMembersBulkOperationResponse)
     _serviceProxy.Execute(retrieveMembersRequest);

 Console.WriteLine("  Contacts with the following GUIDs were distributed to:");

// </snippetdistributecampaignfrommarketinglist5>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist5.vb
' <snippetdistributecampaignfrommarketinglist5>


	' Retrieve the members that were distributed to.
          Dim retrieveMembersRequest =
              New RetrieveMembersBulkOperationRequest With
              {
                  .BulkOperationId = distributeResponse.BulkOperationId,
                  .BulkOperationSource = CInt(Fix(BulkOperationSource.CampaignActivity)),
                  .EntitySource = CInt(Fix(EntitySource.Contact)),
                  .Query = New QueryExpression(Contact.EntityLogicalName)
              }

          Dim retrieveMembersResponse =
              CType(_serviceProxy.Execute(retrieveMembersRequest), 
                  RetrieveMembersBulkOperationResponse)

	Console.WriteLine("  Contacts with the following GUIDs were distributed to:")

' </snippetdistributecampaignfrommarketinglist5>